apps/public-docsite-v9/package.json

```diff
@@ -27,10 +27,8 @@
     "@fluentui/react": "^8.105.2",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 154 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26605&commitId=fc001350d4fb998f8a71a88e6f5f3cf5d75c38b4&env=prod)

✅ There was 77 screenshots added, 77 screenshots removed, 1755 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 154 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Converged.F - Dark Mode.default.chromium.png|0| Added|
|Button Converged.F - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.F - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.F - High Contrast.default.chromium.png|0| Added|
|Button Converged.F - High Contrast.hover.chromium.png|0| Added|
|Button Converged.F - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.I - Dark Mode.default.chromium.png|0| Added|
|Button Converged.I - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.I - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.I - High Contrast.default.chromium.png|0| Added|
|Button Converged.I - High Contrast.hover.chromium.png|0| Added|
|Button Converged.I - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.N - Dark Mode.default.chromium.png|0| Added|
|Button Converged.N - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.N - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.N - High Contrast.default.chromium.png|0| Added|
|Button Converged.N - High Contrast.hover.chromium.png|0| Added|
|Button Converged.N - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.O - Dark Mode.default.chromium.png|0| Added|
|Button Converged.O - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.O - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.O - High Contrast.default.chromium.png|0| Added|
|Button Converged.O - High Contrast.hover.chromium.png|0| Added|
|Button Converged.O - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.T - Dark Mode.default.chromium.png|0| Added|
|Button Converged.T - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.T - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.T - High Contrast.default.chromium.png|0| Added|
|Button Converged.T - High Contrast.hover.chromium.png|0| Added|
|Button Converged.T - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.u - Dark Mode.default.chromium.png|0| Added|
|Button Converged.u - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.u - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.u - High Contrast.default.chromium.png|0| Added|
|Button Converged.u - High Contrast.hover.chromium.png|0| Added|
|Button Converged.u - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.u - RTL.default.chromium.png|0| Added|
|Button Converged.u - RTL.hover.chromium.png|0| Added|
|Button Converged.u - RTL.pressed.chromium.png|0| Added|
|Button Converged.x - Dark Mode.default.chromium.png|0| Added|
|Button Converged.x - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.x - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.x - High Contrast.default.chromium.png|0| Added|
|Button Converged.x - High Contrast.hover.chromium.png|0| Added|
|Button Converged.x - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.y - Dark Mode.default.chromium.png|0| Added|
|Button Converged.y - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.y - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.y - High Contrast.default.chromium.png|0| Added|
|Button Converged.y - High Contrast.hover.chromium.png|0| Added|
|Button Converged.y - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.z - Dark Mode.default.chromium.png|0| Added|
|Button Converged.z - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.z - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.z - High Contrast.default.chromium.png|0| Added|
|Button Converged.z - High Contrast.hover.chromium.png|0| Added|
|Button Converged.z - High Contrast.pressed.chromium.png|0| Added|
|CompoundButton Converged.m - RTL.default.chromium.png|0| Added|
|CompoundButton Converged.m - RTL.pressed.chromium.png|0| Added|
|MenuButton Converged.z - RTL.default.chromium.png|0| Added|
|MenuButton Converged.z - RTL.hover.chromium.png|0| Added|
|MenuButton Converged.z - RTL.pressed.chromium.png|0| Added|
|ToggleButton Converged.m - Dark Mode.default.chromium.png|0| Added|
|ToggleButton Converged.m - Dark Mode.hover.chromium.png|0| Added|
|ToggleButton Converged.m - Dark Mode.pressed.chromium.png|0| Added|
|ToggleButton Converged.m - High Contrast.default.chromium.png|0| Added|
|ToggleButton Converged.m - High Contrast.hover.chromium.png|0| Added|
|ToggleButton Converged.m - High Contrast.pressed.chromium.png|0| Added|
|ToggleButton Converged.m - RTL.default.chromium.png|0| Added|
|ToggleButton Converged.m - RTL.hover.chromium.png|0| Added|
|ToggleButton Converged.m - RTL.pressed.chromium.png|0| Added|
|ToggleButton Converged.w - Dark Mode.default.chromium.png|0| Added|
|ToggleButton Converged.w - Dark Mode.hover.chromium.png|0| Added|
|ToggleButton Converged.w - Dark Mode.pressed.chromium.png|0| Added|
|ToggleButton Converged.w - High Contrast.default.chromium.png|0| Added|
|ToggleButton Converged.w - High Contrast.hover.chromium.png|0| Added|
|ToggleButton Converged.w - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.Default - Dark Mode.default.chromium.png|0| Removed|
|Button Converged.Default - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Default - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Default - High Contrast.default.chromium.png|0| Removed|
|Button Converged.Default - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Default - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Default - RTL.default.chromium.png|0| Removed|
|Button Converged.Default - RTL.hover.chromium.png|0| Removed|
|Button Converged.Default - RTL.pressed.chromium.png|0| Removed|
|Button Converged.Disabled - Dark Mode.default.chromium.png|0| Removed|
|Button Converged.Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Disabled - High Contrast.default.chromium.png|0| Removed|
|Button Converged.Disabled - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Disabled - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Outline Disabled - Dark Mode.default.chromium.png|0| Removed|
|Button Converged.Outline Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Outline Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Outline Disabled - High Contrast.default.chromium.png|0| Removed|
|Button Converged.Outline Disabled - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Outline Disabled - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Primary - Dark Mode.default.chromium.png|0| Removed|
|Button Converged.Primary - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Primary - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Primary - High Contrast.default.chromium.png|0| Removed|
|Button Converged.Primary - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Primary - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Primary Disabled - Dark Mode.default.chromium.png|0| Removed|
|Button Converged.Primary Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Primary Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Primary Disabled - High Contrast.default.chromium.png|0| Removed|
|Button Converged.Primary Disabled - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Primary Disabled - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Subtle - Dark Mode.default.chromium.png|0| Removed|
|Button Converged.Subtle - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Subtle - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Subtle - High Contrast.default.chromium.png|0| Removed|
|Button Converged.Subtle - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Subtle - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Subtle Disabled - Dark Mode.default.chromium.png|0| Removed|
|Button Converged.Subtle Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Subtle Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Subtle Disabled - High Contrast.default.chromium.png|0| Removed|
|Button Converged.Subtle Disabled - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Subtle Disabled - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Transparent - Dark Mode.default.chromium.png|0| Removed|
|Button Converged.Transparent - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Transparent - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Transparent - High Contrast.default.chromium.png|0| Removed|
|Button Converged.Transparent - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Transparent - High Contrast.pressed.chromium.png|0| Removed|
|Button Converged.Transparent Disabled - Dark Mode.default.chromium.png|0| Removed|
|Button Converged.Transparent Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button Converged.Transparent Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button Converged.Transparent Disabled - High Contrast.default.chromium.png|0| Removed|
|Button Converged.Transparent Disabled - High Contrast.hover.chromium.png|0| Removed|
|Button Converged.Transparent Disabled - High Contrast.pressed.chromium.png|0| Removed|
|CompoundButton Converged.Default - RTL.default.chromium.png|0| Removed|
|CompoundButton Converged.Default - RTL.pressed.chromium.png|0| Removed|
|MenuButton Converged.Default - RTL.default.chromium.png|0| Removed|
|MenuButton Converged.Default - RTL.hover.chromium.png|0| Removed|
|MenuButton Converged.Default - RTL.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Default - Dark Mode.default.chromium.png|0| Removed|
|ToggleButton Converged.Default - Dark Mode.hover.chromium.png|0| Removed|
|ToggleButton Converged.Default - Dark Mode.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Default - High Contrast.default.chromium.png|0| Removed|
|ToggleButton Converged.Default - High Contrast.hover.chromium.png|0| Removed|
|ToggleButton Converged.Default - High Contrast.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Default - RTL.default.chromium.png|0| Removed|
|ToggleButton Converged.Default - RTL.hover.chromium.png|0| Removed|
|ToggleButton Converged.Default - RTL.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Primary - Dark Mode.default.chromium.png|0| Removed|
|ToggleButton Converged.Primary - Dark Mode.hover.chromium.png|0| Removed|
|ToggleButton Converged.Primary - Dark Mode.pressed.chromium.png|0| Removed|
|ToggleButton Converged.Primary - High Contrast.default.chromium.png|0| Removed|
|ToggleButton Converged.Primary - High Contrast.hover.chromium.png|0| Removed|
|ToggleButton Converged.Primary - High Contrast.pressed.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>